### PR TITLE
Add IDE support

### PR DESCRIPTION
**Pull request by adelf**
This little config will help Laravel Idea 5.0 complete components and their parameters.

Documentation for this `ide.json` feature isn't ready yet, but I just want to close this - https://github.com/laravel-idea/plugin/issues/285 :)

image

### CODE REVIEW

File: ide.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://laravel-ide.com/schema/laravel-ide-v2.json",
+    "blade": {
+        "components": {
+            "fromConfig": [
+                {
+                    "file": "/config/proton.php",
```
Comment (dyanakiev) [inline ide.json:7]: This should be `"/config/form-components.php"` the package doesn't publish its config as `proton.php`

**Review by dyanakiev — CHANGES_REQUESTED** [review]

File: ide.json (hunk 1 above)
Comment (adelf) [reply ide.json]: Fixed. Thanks.

**Comment by dyanakiev** [comment]
Hey @adelf, any reason why ide.json is deleted?

**Comment by adelf** [comment]
@dyanakiev seems, the package creators don't need it) for some reason)

**Pull request closed without merging**